packages/compose/README.md

```diff
@@ -2,6 +2,51 @@
 
 The `compose` package is a collection of handy [Higher Order Components](https://facebook.github.io/react/docs/higher-order-components.html) (HOCs) you can use to wrap your WordPress components and provide some basic features like: state, instance id, pure...
 
+The **compose** function is an alias to [flowRight](https://lodash.com/docs/#flowRight) from Lodash. It comes from functional programming world and allows to compose any number of functions. An example that illustrates it for two functions:
```

Comment:
```suggestion
The `compose` function is an alias to [flowRight](https://lodash.com/docs/#flowRight) from Lodash. It comes from functional programming, and allows you to compose any number of functions. You might also think of this as layering functions; `compose` will execute each function in sequence, passing the output of the function to the next function.

An example that illustrates it for two functions:
```

Reply:
I tried to make this a little clearer for less technical WordPressers. I think my description is correct but I'd love an audit :)

Reply:
Updated in #13504